drivers/net/ethernet/stmicro/stmmac/stmmac_main.c

```diff
@@ -1658,6 +1658,8 @@ static int stmmac_init_dma_engine(struct stmmac_priv *priv)
 	if (priv->extend_desc && (priv->mode == STMMAC_RING_MODE))
 		atds = 1;
 
+	msleep(1500);
```

Comment:
Please cover such changes in main driver by config flag `JLSEMI_JL2XX1_PHY`

Reply:
Let me try to remove the stmmac sections and do some tests to verify if it is really needed